packages/core-components/src/setupTests.ts

```diff
@@ -15,3 +15,4 @@
  */
 
 import '@testing-library/jest-dom';
+import 'jest-localstorage-mock';
```

Comment:
Is this really needed? Doesn't look like any other package use this while they still interact with localStorage in tests

Reply:
this is still needed. take `timestampStore.test.ts` as an example. This library provides an auto mock for the localStorage API.
the way I'm setting it up is described here https://github.com/clarkbw/jest-localstorage-mock#setup-file

Reply:
I was able to remove it :) thanks